lib/facter/resolvers/load_averages.rb

```diff
@@ -3,6 +3,7 @@
 module Facter
   module Resolvers
     class LoadAverages < BaseResolver
+      @log = Facter::Log.new(self)
```

Comment:
You don't need to define a logger, the base resolver will automatically create one for you. You can use it directly via 
`log.debug(...)`